### DISCUSSION TITLE
SASS does not process styles

### DISCUSSION THREAD

**Discussion by zgoda** [preactjs/wmr#791] [Q&A] [answered]
However strange this may sound, in newly initialised project `build` does not process sass/scss files **at all**. Sass is definitely installed:

```
├── modern-normalize@1.1.0
├── mustard-ui@1.0.4
├── preact-iso@2.0.3
├── preact@10.5.14
├── sass@1.37.5
└── wmr@3.4.1
```

Content is organised as follows:

```
$ tree public
public
├── assets
│   └── fonts
│       ├── lato-v20-latin-ext_latin-italic.woff
│       ├── lato-v20-latin-ext_latin-italic.woff2
│       ├── lato-v20-latin-ext_latin-regular.woff
│       └── lato-v20-latin-ext_latin-regular.woff2
├── index.html
├── index.js
└── style
    ├── fonts.scss
    ├── main.scss
    └── modern-normalize.scss
```

And `build` produces following artifacts:

```
$ tree dist
dist
├── assets
│   ├── fonts
│   │   ├── lato-v20-latin-ext_latin-italic.woff
│   │   ├── lato-v20-latin-ext_latin-italic.woff2
│   │   ├── lato-v20-latin-ext_latin-regular.woff
│   │   └── lato-v20-latin-ext_latin-regular.woff2
│   └── style
│       └── main.d4b979f9.scss
├── chunks
│   └── prerender.6981c0cc.js
├── index.ca23a966.js
├── index.html
└── style
    ├── fonts.scss
    └── modern-normalize.scss
```

The content of `assets/style/main.xxx.scss` is only minified, as if it was not passed to sass at all. Of course, the build result is broken.

Has anyone encountered such weirdness?

**Comment by JoviDeCroock** [comment, marked as answer]
Where are you requiring sass from? If it's the `index.html` I recently made a fix for that and is up for publishing https://github.com/preactjs/wmr/pull/789

**Reply by zgoda** [reply]
In `public/index.html`:

```
<link rel="stylesheet" href="/style/main.scss" />
```

I recall this worked earlier?

**Reply by JoviDeCroock** [reply]
This has actually never worked 😅 but solved with that PR, from JS it will just work

**Reply by zgoda** [reply]
Huh, my memory must be failing me. Thanks!
